src/csvcubed/utils/sparql_handler/csvw_inspector.py

```diff
@@ -48,6 +54,26 @@ def csvw_type(self) -> CSVWType:
                 "The input metadata is invalid as it is not a data cube or a code list."
             )
 
+    @cached_property
+    def _table_schema_properties(self) -> TableSchemaPropertiesResult:
+        """
+        Cached property for the select_table_schema_properties query that stores the query's results.
+        """
+        result = select_table_schema_properties(self.rdf_graph)
+        return result
+
+    @cached_property
+    def _codelist_primary_key_by_csv_url(self) -> PrimaryKeyColNamesByDatasetUrlResult:
+        """
+        Cached property for the select_primary_key_col_names_by_csv_url query that stores the query's results.
+        """
+        csv_url: str
+        result = select_primary_key_col_names_by_csv_url(
```

Comment:
Again, I think we should have this information for each of the csvs present in the CSV-W. I think we should try to combine this query into `_table_schema_properties`.

To be honest, I think we should rename `_table_schema_properties` to `_tables` since it's effectively listing all of the tables in our CSV-W (with the additional information).